spec/std/int_spec.cr

```diff
@@ -385,16 +385,32 @@ describe "Int" do
     UInt64.new(1).should eq(1)
   end
 
-  it "divides negative numbers" do
-    (7 / 2).should eq(3)
-    (-7 / 2).should eq(-4)
-    (7 / -2).should eq(-4)
-    (-7 / -2).should eq(3)
+  describe "arithmetic division /" do
+    it "divides negative numbers" do
+      (7 / 2).should eq(3.5)
+      (-7 / 2).should eq(-3.5)
+      (7 / -2).should eq(-3.5)
+      (-7 / -2).should eq(3.5)
+
+      (6 / 2).should eq(3.0)
+      (-6 / 2).should eq(-3.0)
+      (6 / -2).should eq(-3.0)
+      (-6 / -2).should eq(3.0)
+    end
 
-    (6 / 2).should eq(3)
-    (-6 / 2).should eq(-3)
-    (6 / -2).should eq(-3)
-    (-6 / -2).should eq(3)
+    it "does not raises when divides by zero" do
+      (1 / 0).should eq(Float64::INFINITY)
+      (4 / 2).should eq(2)
```

Comment:
What's the purpose of this example?

Reply:
leftovers from refactors, removing it since it's covered by others.